docs/jetstream/operations.md

```diff
@@ -38,17 +38,42 @@ In addition to running experiment analyses in parallel, [Dask](https://dask.org/
 
 The [`dask.delayed interface`](https://docs.dask.org/en/latest/delayed.html) is used to turn the functions executing these steps into tasks that are added to a task graph which executes these steps in parallel. Dask is configured to use as many cores as are available on the machine by default, with 1 worker for each core. [Multi-threading being avoided, instead processes are used](https://docs.dask.org/en/latest/scheduling.html#local-threads) since the code is dominated by Python code, otherwise there wouldn't be any speedup due Python's Global Interpreter Lock. To manually restrict the number of processes, the `JETSTREAM_PROCESSES` environment variable can be used.
 
-## Setup
+## Installation
 
 Jetstream is executed on the `jetstream` Kubernetes cluster in the `moz-fx-data-experiments` project which is set up following [Argo's installation guide](https://github.com/argoproj/argo/blob/master/docs/quick-start.md):
 * When creating or re-creating the cluster, BigQuery and Compute Engine read/write permissions need to be enabled
 * Installing Argo:
 	* Create an `argo` namespace: `kubectl create ns argo`
-	* Install commonly used components: `kubectl apply -n argo -f https://raw.githubusercontent.com/argoproj/argo/stable/manifests/quick-start-postgres.yaml`
+	* Install commonly used components: `kubectl apply -n argo -f https://github.com/argoproj/argo-workflows/releases/download/v3.4.5/install.yaml`
 	* Create new `clusterrole`: `kubectl create rolebinding default-admin --clusterrole=admin --serviceaccount=argo:default --namespace=argo`
 * The [`jetstream` DAG in Airflow](https://github.com/mozilla/telemetry-airflow/blob/master/dags/jetstream.py) triggers the `run-argo` job daily and either requires Compute Engine API access or the parameters `cluster_ip` and `cluster_cert` need to be provided
 	* Currently the Airflow cluster does not have Compute Engine API access, so the cluster IP and certificate are stored as secrets and used for running Jetstream
 
+## Argo Workflow UI
+
+Argo provides a Web UI to access running workflows. Users need to authenticate using a Bearer token:
+* Get Bearer token and copy: `gcloud container clusters get-credentials jetstream --region=us-central1-a && kubectl -n argo exec $(kubectl get pod -n argo -l 'app=argo-server' -o jsonpath='{.items[0].metadata.name}') -- argo auth token` 
```

Comment:
I had to add the `CLOUDSDK_CORE_PROJECT=moz-fx-data-experiments` parameter at the beginning to get the command to work (guess I was defaulted to the `moz-fx-data-bq-data-science` project). Maybe we can add this as a note in a sub-bullet point

Reply:
Should we note that warnings about the old auth scheme being deprecated are safe to ignore?